CMakeLists.txt

```diff
@@ -47,18 +48,33 @@ endif()
 
 add_compile_definitions(NITRO_VERSION="${NITRO_VERSION}")
 
-add_subdirectory(llama.cpp)
-add_executable(${PROJECT_NAME} main.cc)
+
+
+
+
+
+set(CMAKE_PREFIX_PATH ${CMAKE_CURRENT_SOURCE_DIR}/build_deps/_install
+) # This is the critical line for installing another package
+
+
+
+
+# third party import path
+# include_directories(build_deps/_install/include)
+# # third party objects
+# link_directories(build_deps/lib)
 
 # ##############################################################################
 # If you include the drogon source code locally in your project, use this method
 # to add drogon add_subdirectory(nitro_deps)
 # target_link_libraries(${PROJECT_NAME} PRIVATE nitro_deps)
 #
 # and comment out the following lines
-find_package(Drogon CONFIG REQUIRED)
-target_link_libraries(${PROJECT_NAME} PRIVATE Drogon::Drogon common llava
-                                              ${CMAKE_THREAD_LIBS_INIT})
+# find_package(Drogon CONFIG REQUIRED)
+# target_link_libraries(${PROJECT_NAME} PRIVATE Drogon::Drogon common llava
+#                                               ${CMAKE_THREAD_LIBS_INIT})
```

Comment:
what was replaced with in this section

Reply:
decalred in src here: https://github.com/janhq/nitro/pull/325/files#diff-148715d6ea0c0ea0a346af3f6bd610d010d490eca35ac6a9b408748f7ca9e3f4